### ISSUE TITLE
Problem about Constructing Boxplot by Bokeh

### ISSUE THREAD

**Issue by richardshiii** [neuniversity/ALY6140#17]
Hi all, 
I am currently stuck on the third problem of week 4 assignment. The problem asks us to construct boxplots using bokeh. Resources I found online contain large amount of coding. I tried to go through all of it, but I do not believe it is the most effective solution for this problem. If anyone is experienced with bokeh or finds other resources, please let me know. Thanks a lot!

Link to online resources: https://bokeh.pydata.org/en/latest/docs/gallery/boxplot.html

**Comment by siuol456** [comment]
Hi,
Thank you for the sharing. I think it is really helpful. On the other hand, when I am reading the problem, I feel confused. Instead of a boxplot, do you think it should be a barplot, since that the question is saying, "plot that shows 'average bill' amount by day of week"?

**Comment by siuol456** [comment]
Hi,
I found this website that use bokeh as an extension. http://holoviews.org/gallery/demos/bokeh/boxplot_chart.html

**Comment by Jin-pengSong** [comment]
I also have the same concern that the coding in https://bokeh.pydata.org/en/latest/docs/gallery/boxplot.html is too much. Have you got other resources?

**Comment by pr24** [comment]
yes, even I am facing the same issue.there were lot of coding contain if we want to create the box plot using bokeh.

Do we need to install some module for that??? like bkcharts or bokeh,plotting??

**Comment by Jin-pengSong** [comment]
I also use the similar code for this question. However, while I run my
code, it shows this kind of information instead of the picture you showed.
